scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -88,6 +88,15 @@ abstract class H2OMOJOModel
     new IntParam(this, "numberOfCrossValidationModels", "Number of cross validation models.")
   protected var crossValidationModels: Array[H2OMOJOModel] = null
 
+  protected final val startTime: LongParam =
+    new LongParam(this, "startTime", "Start time in milliseconds")
+  protected final val endTime: LongParam =
+    new LongParam(this, "endTime", "End time in milliseconds")
+  protected final val runTime: LongParam =
+    new LongParam(this, "runTime", "Runtime in milliseconds")
+  protected final val defaultThreshold: DoubleParam =
+    new DoubleParam(this, "defaultThreshold", "Default threshold used for predictions")
```

Comment:
Default threshold used for predictions of classification models?

Reply:
Sounds good, I'm gonna update the documentation string.